var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -25,6 +25,7 @@ class Dealii(CMakePackage, CudaPackage):
     generator = 'Ninja'
 
     version('master', branch='master')
+    version('9.3.2', branch='dealii-9.3')
```

Comment:
We're currently preparing the next point release, so I'd prefer it if we did not target the git branch. We're currently running tests on the branch, which should be complete in the next day or two. I'll provide you a `sha` when it's done.

Reply:
Sure - this is a temporary place-holder (for testing) until deal@9.3.2 is released.

Once released - it can be a separate PR to spack:develop. And once it gets merged in - this branch can be rebased (to remove this change)

Reply:
I've opened #27303 for this change.